snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/rf2/exporter/Rf2RefSetExporter.java

```diff
@@ -257,7 +257,15 @@ protected SnomedRefSetMemberSearchRequestBuilder createSearchRequestBuilder() {
 					// Append extra columns using the properties map
 					final Map<String, Object> properties = member.getProperties();
 					for (final String extraColumn : extraColumns) {
-						builder.add(toColumn(properties.get(extraColumn)));
+						Object property = properties.get(extraColumn);
+						if (extraColumn.equals(SnomedRf2Headers.FIELD_QUERY)) {
+							String query = (String) property;
+							query = query.replace("\r", " ");
```

Comment:
A simplified regex would be better here to reduce multiple control characters into a single space: `replaceAll("[\r\n]+", " ")`.

Reply:
Also, move the logic to the `toColumn()` method.